workitem/workitem_repository.go

```diff
@@ -687,39 +690,58 @@ func (r *GormWorkItemRepository) Fetch(ctx context.Context, spaceID uuid.UUID, c
 // 		WHERE (iterations.space_id = '33406de1-25f1-4969-bcec-88f29d0a7de3'
 // 		and work_items.deleted_at IS NULL) GROUP BY IterationId
 func (r *GormWorkItemRepository) GetCountsPerIteration(ctx context.Context, spaceID uuid.UUID) (map[string]WICountsPerIteration, error) {
-	var res []WICountsPerIteration
-	db := r.db.Table("work_items").Select(`iterations.id as IterationId, count(*) as Total,
-				count( case fields->>'system.state' when 'closed' then '1' else null end ) as Closed`).Joins(`left join iterations
-				on fields@> concat('{"system.iteration": "', iterations.id, '"}')::jsonb`).Where(`iterations.space_id = ?
-				and work_items.deleted_at IS NULL`, spaceID).Group(`IterationId`).Scan(&res)
+	var itrs []*iteration.Iteration
+	db := r.db.Debug()
+	db = db.Table("iterations").Select("*").Where("space_id = ? AND deleted_at IS NULL", spaceID).Scan(&itrs)
```

Comment:
I don't think you need the `deleted_at IS NULL` part. This should be added by GORM automatically unless you run `db.Unscoped().Table(....)`